### PR TITLE
#27 - Correction of the Idempotency of the tasks

### PR DESCRIPTION
**Pull request by Jalkar**
Bugfix of issue #27 
Correction of the idempotency based on the following findings :
- https://symfonycasts.com/screencast/ansible/idempotency-changed-when (for .bashrc & .bashprofile)
- https://nickjanetakis.com/blog/debugging-an-idempotency-related-user-group-issue-with-ansible (for user creation)

### CODE REVIEW

File: roles/splunk/tasks/install_splunk.yml [hunk 1]
```diff
@@ -13,6 +13,7 @@
         name: "{{ splunk_nix_user }}"
         groups: "{{ splunk_nix_group }}"
         home: "{{ splunk_home }}"
+        append: true
```
Comment (mason-splunk) [inline roles/splunk/tasks/install_splunk.yml:16]: Good catch

File: roles/splunk/tasks/configure_bash.yml [hunk 2]
```diff
@@ -10,3 +10,4 @@
   loop:
     - { bashtemplate: 'bashrc.j2', bashfilepath: '.bashrc' }
     - { bashtemplate: 'bash_profile.j2', bashfilepath: '.bash_profile' }
+  changed_when: False
```
Comment (mason-splunk) [inline roles/splunk/tasks/configure_bash.yml:13]: This doesn't really fix idempotency for this task. It just tells Ansible to report that nothing changed even if it did make changes to these files.

Comment (mason-splunk) [reply roles/splunk/tasks/configure_bash.yml:13]: I tested solely the `append: true` part of this PR without the change to `configure_bash.yml` and was able to confirm that it resolves the idempotency issue for both tasks. Please remove this change from `configure_bash.yml` and we'll get your PR merged. Thank you!

Comment (Jalkar) [reply roles/splunk/tasks/configure_bash.yml:13]: oups, it tested but obviously i'm unable to read "OK" vs "CHANGED" 👎 
I'll update the PR

Comment (Jalkar) [reply roles/splunk/tasks/configure_bash.yml:13]: And in fact without my changes, i don't reproduce the issue for the `configure_bash.yml`
First execution : 
image

Second execution : 
image

**Pull request merged**